crates/holochain/src/core/ribosome/host_fn/capability_claims.rs

```diff
@@ -1,14 +1,13 @@
 use crate::core::ribosome::error::RibosomeResult;
 use crate::core::ribosome::CallContext;
 use crate::core::ribosome::RibosomeT;
-use holochain_types::prelude::*;
 use std::sync::Arc;
 
 /// lists all the local claims filtered by tag
 pub fn capability_claims(
     _ribosome: Arc<impl RibosomeT>,
     _call_context: Arc<CallContext>,
-    _input: CapabilityClaimsInput,
-) -> RibosomeResult<CapabilityClaimsOutput> {
+    _input: (),
```

Comment:
see previous suggestion for an empty enum

Reply:
unimplemented